From 6ab86b9503cdcac775d3313e60d31fa8c031aa98 Mon Sep 17 00:00:00 2001
From: Karthikeyan K <karthikk@habana.ai>
Date: Wed, 28 Sep 2022 19:16:17 +0300
Subject: [PATCH] [SW-106171] gaudi3: tpm: update baud rate for onix

Updating the tpm spi baudrate to 4 in onix, as per recommendation
in tpm code review.

Change-Id: I7874274840969fe2ce8cfe5ad429cdb6d33c6ce7
---

diff --git a/common/spi.c b/common/spi.c
index 4b84b6a..7cbec43 100644
--- a/common/spi.c
+++ b/common/spi.c
@@ -346,10 +346,10 @@
 	spi_writel(ctx, 0, TXFTLR);
 }
 
-#define DW_SPI_BAUD_MIN		2
+#define DW_SPI_BAUD_RATE		4
 __weak uint32_t hl_spi_baudr_get()
 {
-	return DW_SPI_BAUD_MIN;
+	return DW_SPI_BAUD_RATE;
 }
 
 void spi_init(struct spi_ctx *ctx)
